app/views/posts/index.html.erb

```diff
@@ -1 +1,39 @@
-<h1>Display All the posts for a specific user</h1>
\ No newline at end of file
+<%= stylesheet_link_tag "post_index.css" %>
+<h1>Display All the posts for a specific user</h1>
+<div class='user'>
+        <div class="face-container">
+          <img 
+          class="face" 
+          src=<%= @user.Photo %>
+          alt="face" />
+        </div>
+        <div class="details">
+            <h1><%= @user.Name %></h1>
+            <h3>Number of posts : <%= @user.PostsCounter %></h3>
+        </div>
+    </div>
+    <% @posts.each do |post| %>
+      <a href="/users/<%= @user.id %>/posts/<%= post.id %>">
+      <div>
+          <div class="post">
+            <h3><%= post.Title %></h3>
+            <p><%= post.Text %></p>
+            
+          </div>
+          
+            <div class="comment-details">
+            <% post.comments.each do |comment| %>
```

Comment:
- [ ] Hi @yasinabdmahmood, let's retrieve and render the five most recent comments of each post. It is mentioned in the requirements. The piece of code below does the job I think.

```ruby
 <% post.retrieve_recent_comments.each do |comment| %>
```

- Only post show page should display all these comments

Reply:
Well resolved 👍🏽